packages/LGvscodeExt/syntaxes/lg.tmLanguage.json

```diff
@@ -80,7 +80,8 @@
 			]
 		},
 		"params": {
-			"match": "\\((?:[^)(]+|\\((?:[^)(]+|\\([^)(]*\\))*\\))*\\)",
+			"begin": "\\(",
```

Comment:
So, we don't use `match` any more, is this begin end can cross line or not?

Reply:
yes. it can cross lines.

Reply:
we don't want cross lines, right?

Reply:
Let's leave this to next PR maybe